src/admin-views/help-hub/resources/resource_template.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * The template that displays the resources tab on the help page.
+ *
+ * @var Tribe__Main $main             The main common object.
+ * @var Hub         $help_hub         The Help Hub class.
+ * @var string      $template_variant The template variant, determining which template to display.
+ * @var array       $section          The section to display.
+ */
+
+// Skip FAQ sections and sections without required data.
+if (
+	empty( $section ) ||
+	! is_array( $section ) ||
+	empty( $section['title'] ) ||
+	empty( $section['links'] ) ||
+	! is_array( $section['links'] ) ||
+	strtolower( $section['title'] ) === 'faq'
```

Comment:
notes:
The first empty check is redundant. The first array check may be as well, since if it's not an array `empty( $section['title'] )` will fail.

Reply:
```suggestion
	( empty( $section['links'] ) || ! is_array( $section['links'] ) )
	|| ( empty( $section['title'] ) || strtolower( $section['title'] ) === 'faq' )
```
Above plus some logical grouping, probably needs formatting.